src/current/v23.2/known-limitations.md

```diff
@@ -30,13 +30,15 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
 
     [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/84680)
 
-### Limitations for `SELECT FOR UPDATE`
+### Limitations for `SELECT ... FOR UPDATE` and `SELECT ... FOR SHARE`
 
-- [`SELECT FOR UPDATE`]({% link {{ page.version.version }}/select-for-update.md %}) places locks on each key scanned by the base index scan. This means that even if some of those keys are later filtered out by a predicate which could not be pushed into the scan, they will still be locked.
+The following limitations reflect the default behavior under [`SERIALIZABLE`]({% link {{ page.version.version }}/demo-serializable.md %}) isolation. They are fixed by setting the `enable_durable_locking_for_serializable` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) to `true`. These limitations do **not** apply to [`READ COMMITTED`]({% link {{ page.version.version }}/read-committed.md %}) transactions.
+
+- [`SELECT ... FOR UPDATE`]({% link {{ page.version.version }}/select-for-update.md %}) and [`SELECT ... FOR SHARE`]({% link {{ page.version.version }}/select-for-update.md %}) place locks on each key scanned by the base index scan. This means that even if some of those keys are later filtered out by a predicate which could not be pushed into the scan, they will still be locked.
```

Comment:
I added a paragraph stating that both of the limitations in this section apply to Serializable and are fixed with the cluster setting to enable durable locks. Is this particular limitation gone regardless? @michae2, I'd also appreciate your eyes on how I've updated this section

Reply:
I like your approach to keep the limitations, but preface them, since they are still true under serializable.

The first two limitations are actually fixed by `optimizer_use_lock_op_for_serializable` rather than `enable_durable_locking_for_serializable`. (The third is fixed by durable locking.)